stanford/sandbox.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Configure sandbox
```

Comment:
This is mostly pulled right from `playbooks/edx_sandbox.yml`. 
We may be better served to symlink to there and fork lines as needed.